libs/runtime/ebpf_hash_table.h

```diff
@@ -196,8 +196,7 @@ extern "C"
      * @param[out] next_key_pointer Pointer to next key if one exists.
      * @param[out] next_value If non-NULL, returns the next value if it exists.
      * @retval EBPF_SUCCESS The operation was successful.
-     * @retval EBPF_NO_MORE_KEYS No keys exist in the hash table that
-     * are lexicographically after the specified key.
+     * @retval EBPF_NO_MORE_KEYS No keys exist in the hash table.
```

Comment:
This message seems wrong.  Perhaps:
```suggestion
     * @retval EBPF_NO_MORE_KEYS No more keys exist in the hash table.
```